src/Parser/Monk/Windwalker/Modules/Spells/StormEarthAndFire.js

```diff
@@ -74,31 +83,87 @@ class StormEarthAndFire extends Analyzer{
+            <SpellIcon id={SPELLS.RISING_SUN_KICK.id} noLink /> Rising Sun Kick
+          </SpellLink>
+        </div>
+        <div className="flex-sub text-right">
+          {this.risingSunKicks}/{this.castCount * (2 + this.drinkingHornCover) + 1}
+        </div>
+      </div>
+    );
+  }
 
-  statistic() {
-   const icon = this.combatants.selected.hasTalent(SPELLS.SERENITY_TALENT.id) ? SPELLS.SERENITY_TALENT : SPELLS.STORM_EARTH_AND_FIRE_CAST;
-   const drinkingHornCover = this.combatants.selected.hasWrists(ITEMS.DRINKING_HORN_COVER.id) ? 1 : 0;
-   const castCount = this.abilityTracker.getAbility(icon.id).casts;
-   return (
-     <StatisticBox
-       icon={<SpellIcon id={icon.id} />}
-       label={
-          <div>
-           During your {castCount} {icon.name}s you cast:
-          <li>{this.risingSunKicks}/{castCount * (2 + drinkingHornCover) + 1} Rising Sun Kicks</li>
-          <li>{this.fistsOfFuries}/{castCount + drinkingHornCover} Fists of Furies</li>
-          <li>{this.strikeOfTheWindlords}/{castCount} Strikes of the Windlord</li>
-              {this.whirlingDragonPunches > 0
-              ? <li>{this.whirlingDragonPunches}/{castCount} Whirling Dragon Punches</li> : ''
-              }
+  fistsofFuryStatistic() {
+    return (
+      <div className="flex">
+        <div className="flex-main">
+          <SpellLink id={SPELLS.FISTS_OF_FURY_CAST.id}>
+            <SpellIcon id={SPELLS.FISTS_OF_FURY_CAST.id} noLink /> Fist of Fury
+          </SpellLink>
+        </div>
+        <div className="flex-sub text-right">
+          {this.fistsOfFuries}/{this.castCount + this.drinkingHornCover}
         </div>
-       }
-     />
-   );
+      </div>
+    );
   }
-statisticOrder = STATISTIC_ORDER.CORE(14);
 
+  whirlingDragonPunchStatistic() {
+    if (this.abilityTracker.getAbility(SPELLS.WHIRLING_DRAGON_PUNCH_TALENT.id).casts >0) {
+      return(
+      <div className= "flex" >
+          <div className="flex-main">
+            <SpellLink id={SPELLS.WHIRLING_DRAGON_PUNCH_TALENT.id}>
+              <SpellIcon id={SPELLS.WHIRLING_DRAGON_PUNCH_TALENT.id} noLink /> Whirling Dragon Punch
+          </SpellLink>
+          </div>
+          <div className="flex-sub text-right">
+            {this.whirlingDragonPunches}/{this.castCount}
+          </div>
+      </div>
+    );}
+  }
+
+    strikeoftheWindlordStatistic() {
+      return (
+        <div className="flex">
+          <div className="flex-main">
+            <SpellLink id={SPELLS.STRIKE_OF_THE_WINDLORD.id}>
+              <SpellIcon id={SPELLS.STRIKE_OF_THE_WINDLORD.id} noLink /> Strike of the Windlord
+          </SpellLink>
+          </div>
+          <div className="flex-sub text-right">
+            {this.strikeOfTheWindlords}/{this.castCount}
+          </div>
+        </div>
+      );
+    }
 
+  statistic() {
+    this.castCount = this.abilityTracker.getAbility(this.ability.id).casts; 
+    return (
+      <StatisticsListBox
+        title={
+          <span>
```

Comment:
Please use the `Wrapper` component as a container when the container has no function.